src/commands/misc/flathub.js

```diff
@@ -0,0 +1,45 @@
+const { SearchCommand, Constants, SwitchbladeEmbed } = require('../../')
+const Turndown = require('turndown')
+const turndownService = new Turndown()
+const moment = require('moment')
+
+module.exports = class FlatHubCommand extends SearchCommand {
+  constructor (client) {
+    super({
+      name: 'flathub',
+      aliases: ['flath'],
+      requirements: { apis: ['flathub'] },
+      embedColor: Constants.FLATHUB_COLOR,
+      embedLogoURL: 'https://flathub.org/assets/themes/flathub/flathub-logo.png'
```

Comment:
Please refrain from using third-party websites. At best, [upload the image to Imgur](https://imgur.com)

Reply:
Do you want that i do a pull request for all third-party links ?
There are a lot of links coming from a others websites

Reply:
I meant for static image URLs only. But if you were talking about this as well, then sure!